docs/src/main/sphinx/object-storage/file-system-s3.md

```diff
@@ -28,6 +28,8 @@ support:
   - Required region name for S3.
 * - `s3.path-style-access`
   - Use path-style access for all requests to S3
+* - `s3.exclusive-create`
+  - Whether conditional write is supported by the S3-compatible storage. Defaults to `true`.
```

Comment:
do we need to configure that? Or maybe we can figure out automatically?

Reply:
I don't think that we can figure it out automatically. Basically this works by sending a header `If-none-match: *` which server interprets as exclusive write. There is no way of checking that